internal/resource/config/oauth/accesstokenmapping/oauth_access_token_mappings_resource.go

```diff
@@ -93,7 +93,7 @@ func (r *oauthAccessTokenMappingsResource) Schema(ctx context.Context, req resou
 				Attributes: resourcelink.ToSchemaNoLocation(),
 			},
 			"attribute_sources":              attributesources.ToSchema(0, true, false),
-			"attribute_contract_fulfillment": attributecontractfulfillment.ToSchema(true, false, false),
+			"attribute_contract_fulfillment": attributecontractfulfillment.ToSchema(true, false, true),
```

Comment:
It seems like from the bug description we should be setting a default empty string value for the "value" attribute. Does this method set a default?

Reply:
Added string default